codegen/smithy-aws-ruby-codegen/src/main/resources/ruby/customizations/s3/endpoint_rules.rb

```diff
@@ -1,5 +1,7 @@
+# S3 Endpoint custom functions
 module Endpoints
   # aws.isVirtualHostableS3Bucket(value: string, allowSubDomains: bool) bool
+  # rubocop:disable Style/OptionalBooleanParameter
```

Comment:
Would be nice to fix this one .... I tried

Reply:
Fixed - just removed the default. This matches the endpoints language definitions of the function more closely anyway (which ofc does not define/describe default function values).